programs/range/eigenda/src/main.rs

```diff
@@ -0,0 +1,44 @@
+//! A program to verify a Optimism L2 block STF with Ethereum DA in the zkVM.
+//!
+//! This binary contains the client program for executing the Optimism rollup state transition
+//! across a range of blocks, which can be used to generate an on chain validity proof. Depending on
+//! the compilation pipeline, it will compile to be run either in native mode or in zkVM mode. In
+//! native mode, the data for verifying the batch validity is fetched from RPC, while in zkVM mode,
+//! the data is supplied by the host binary to the verifiable program.
+
+#![no_main]
+sp1_zkvm::entrypoint!(main);
+
+use hokulea_proof::{
+    canoe_verifier::sp1_cc::CanoeSp1CCVerifier, eigenda_blob_witness::EigenDABlobWitnessData,
```

Comment:
NOTE: current EigenDA testnet is on holesky so maybe good to test e2e with Noop verifier first. Request has been made to migrate to sepolia.